pkgs/development/python-modules/pygsl/default.nix

```diff
@@ -0,0 +1,48 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pkgs
+, numpy
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "pygsl";
+  version = "2.3.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-F3m85Bs8sONw0Rv0EAOFK6R1DFHfW4dxuzQmXo4PHfM=";
+  };
+
+  nativeBuildInputs = [
+    pkgs.gsl.dev
+    pkgs.swig
+  ];
+  buildInputs = [
+    pkgs.gsl
+  ];
+  propagatedBuildInputs = [
+    numpy
+  ];
+
+  preCheck = ''
+    # have to move out of the source directory for some reason
```

Comment:
Try setting PYTHONPATH to site-packages in out

Reply:
It seems that `cd tests` suffices.